src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1309,7 +1309,7 @@ public function registerForAutoconfiguration(string $interface)
     /**
      * Registers an attribute that will be used for autoconfiguring annotated classes.
      *
-     * The configurator will receive a ChildDefinition instance, an instance of the attribute and the corresponding \ReflectionClass, in that order.
+     * The configurator will receive a ChildDefinition instance, an instance of the attribute, the corresponding \ReflectionClass, and optional \ReflectionMethod, in that order.
```

Comment:
```
     * The configurator will receive a ChildDefinition instance, the attribute instance, and the corresponding
     * \ReflectionClass|\ReflectionMethod|\ReflectionProperty|\ReflectionParameter, in that order.
```

Reply:
Changed, but `ReflectionParameter` is still not implemented.